dev-util/clue/clue-2.2.0.recipe

```diff
@@ -0,0 +1,55 @@
+SUMMARY="A GUI tracing tool"
+DESCRIPTION="Clue allows inspection on almost all of the haiku native objects."
+HOMEPAGE="https://github.com/HaikuArchives/Clue"
+COPYRIGHT="Jeff Braun and Mathew Hounsell"
+LICENSE="GNU LGPL v2"
+REVISION="1"
+srcGitRev="0d49930871aa9e3c539f094f299d8a1f43150303"
+SOURCE_URI="https://github.com/HaikuArchives/Clue/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="33fa1f1000c62090ed285bc8e9e81f2f1bccc50a0ef2ab4ccaa1c4949f5dcfc1"
+SOURCE_FILENAME="Clue-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="Clue-$srcGitRev"
+PATCHES="clue-$portVersion.patchset"
+
+ARCHITECTURES="x86 x86_gcc2 x86_64"
```

Comment:
The preferred order for [`ARCHITECTURES`](https://github.com/haikuports/haikuports/wiki/HaikuPorter-BuildRecipes#architectures) is:
```
ARCHITECTURES="x86_gcc2 x86 x86_64" 
```
Okay, this is not that important, but if you could put `x86_gcc2` first it would be nice because in most of our recipes `ARCHITECTURES` starts with `x86_gcc2` (or `!x86_gcc2`.) Thanks!